src/multivariate/mvnormal.jl

```diff
@@ -271,6 +278,18 @@ function _rand!(rng::AbstractRNG, d::MvNormal, x::AbstractVector)
     add!(unwhiten!(d.Σ, x), d.μ)
 end
 
+### Affine transformations
+
++(c::AbstractVector, d::MvNormal) = MvNormal(d.μ .+ c, d.Σ)
+
+@inline +(d::MvNormal, c::AbstractVector) = c + d
+
+*(B::AbstractMatrix, d::MvNormal) = MvNormal(B * d.μ, X_A_Xt(d.Σ, B))
+
```

Comment:
Where are `X_A_Xt` and `√quad` coming from? Could we use a qualified name instead? `Module.X_A_Xt`

Reply:
It is coming from `PDMats`. No other symbols from that package are qualified in the code, why should these two be an exception?

Reply:
> No other symbols from that package are qualified in the code

This for a start is bad practice on our side, I wanted to fix it once by explicitly importing things but others contributors were not fond of it.

It is non-trivial that these come from PDMats,.jl unlike, say `PDMat`

Reply:
When I make a PR to a project, I try to follow existing coding style. While I don't necessarily disagree about the style suggestions you are making, I feel that the proper approach is to make these changes in the package first, perhaps clarify coding style (with a link to one of the existing ones like https://github.com/jrevels/YASGuide), and *then* ask for PRs to follow them. 

Conversely, asking for a coding style clearly not followed by a package in PRs is somewhat unfriendly, even when meant well, because it results in endless minor revisions trying to conform to a style that is impossible to predict in advance. I could of course just fix things up, but since I find this tiresome, my policy these days is just to close the PR when this happens.

I am not trying to be difficult here — I am totally fine with people branching and modifying this code, or having it rejected.

Reply:
Again, this is not an issue with the code in this PR, and this comment is minor enough not to be blocking the PR. What we have to keep in mind is that unlike many packages, Distributions is not "owned" by a very core team. If you look at the last commits for maybe one year, many have been done by one-time contributors, or rare contributors.
Lots of parts are really old and keeping track of it all is hard to keep track of everything, some things that were valid for 5 contributors always working on it is not anymore with the new setting, thus trying to switch to more explicit style.
For what I mentioned earlier, this PR https://github.com/JuliaStats/Distributions.jl/pull/778 was the first attempt